/*
 * ElevationChangeDetection.hpp
 *
 *  Created on: Mar 26, 2015
 *      Author: Martin Wermelinger
 *	 Institute: ETH Zurich, Autonomous Systems Lab
 */

#pragma once

namespace elevation_change_detection {

  class ElevationChangeDetection {

  public:
    /*!
    * Constructor.
    */
    ElevationChangeDetection();

    /*!
    * Destructor.
    */
    virtual ~ElevationChangeDetection();

  };

} /* namespace elevation_change_detection*/
